deployments/data101/config/common.yaml

```diff
@@ -91,21 +91,6 @@ jupyterhub:
         c.Spawner.pre_spawn_hook = ensure_db_pvc
     nodeSelector:
       hub.jupyter.org/pool-name: core-pool-2024-05-08
-    config:
```

Comment:
@balajialg this block of code was duplicated, and is defined higher up in the config

Reply:
Ah, got it. Thanks for the fix